File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        
        officer=0
        for CM in range(1,len(nums)):
            if nums[CM]!=nums[CM-1]:
                officer+=1
                nums[officer]=nums[CM]
        return officer +1
            
        

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {

        int officer = 0;
        for(int CM=1;CM<nums.length;CM++)
        {
            if (nums[CM]!=nums[CM-1])
            {
                officer++;
                nums[officer]=nums[CM];
            }
        }
        return officer+1;
    }
}